src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -70,6 +74,7 @@ public JsonAdaptedPerson(Person source) {
         projectStatus = source.getProjectStatus().toString();
         paymentStatus = source.getPaymentStatus().toString();
         clientStatus = source.getClientStatus().toString();
+        deadline = source.getDeadline().value.format(Deadline.INPUT_FORMATTER);
```

Comment:
I could be wrong here but I'm not sure if the format part is necessary considering the other parameters did not require such a checker

Reply:
how about just using `source.getDeadline().toString()`?